src/Matcher.js

```diff
@@ -39,6 +39,16 @@ export default class Matcher {
     });
   }
 
+  joinPath(basePath, path) {
+    if (!path) {
+      return basePath;
+    }
+    if (basePath.charAt(basePath.length - 1) === '/') {
+      return `${basePath}${path}`;
+    }
+    return `${basePath}/${path}`;
```

Comment:
joining `a`, `/b` will give you `a//b`, which is wrong. you probably want

```js
`${basePath}${this.getCanonicalPath(path)}
```

and strip the trailing slash on `basePath` if one is present